packages/imba/bin/imba-create.imba

```diff
@@ -22,9 +22,15 @@ def write-package dir, data
 let imbadir = np.resolve(__dirname,'..')
 let imbapkg = read-package(imbadir) # JSON.parse(nfs.readFileSync(np.resolve(imbadir,'package.json'),'utf-8'))
 
+# NOTE: These must start with `imba-` on Github. These names expect `imba-` to be prepended when resolving the URL.
 const templates = [
-	['base-template', 'Application with client-side scripts'],
-	['electron-template', 'Electron application']
+	['base-template', 'Website with both frontend and backend']
+	['static-base-template', 'Static website without backend']
+	['static-netlify-template', 'Static website without backend, on Netlify']
```

Comment:
Can we merge the 4 templates (gh, netlify, cf and static) into one. It should be the same code that's being built.

Reply:
They would have different onboarding instructions, configs per environment, and potentially differ over time, but this is a fair point. I'll collapse them into one and see if we hit any blockers that would mean it's better to split them out.